scripts/screener/screener.runner.ts

```diff
@@ -109,5 +107,13 @@ export async function screenerRunner(screenerConfigPath) {
       : undefined,
   });
 
-  await notifyIntegration(commit, checkUrl);
+  await notifyIntegration({ commit, url: checkUrl, status: 'in_progress', project: screenerConfig.projectRepo });
+}
+
+export async function cancelScreenerRun(screenerConfigPath) {
+  const screenerConfig: ScreenerRunnerConfig = require(screenerConfigPath) as any;
```

Comment:
can we avoid patterns that wont work for ESM ? I'd suggest to have the config passed as argument rather then dynamic path.

Reply:
I've updated it so that the task requires the config instead and passes that to the runner API